eng/Versions.props

```diff
@@ -63,6 +51,30 @@
     <MicrosoftWindowsDesktopAppPackageVersion>3.0.0-preview8-27919-09</MicrosoftWindowsDesktopAppPackageVersion>
     <MicrosoftWindowsDesktopPackageVersion>$(MicrosoftWindowsDesktopAppPackageVersion)</MicrosoftWindowsDesktopPackageVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- 3.0 Template versions -->
+    <!-- Dependency from https://github.com/dotnet/winforms -->
+    <MicrosoftDotnetWinFormsProjectTemplatesPackageVersion>4.8.0-preview8.19359.1</MicrosoftDotnetWinFormsProjectTemplatesPackageVersion>
```

Comment:
I wonder if these should be versioned as well... 3.0/3.1 etc? Otherwise, what will happen when we ship 3.1?

Reply:
We should not change:

- MicrosoftDotnetWinFormsProjectTemplatesPackageVersion
- MicrosoftDotNetWpfProjectTemplatesPackageVersion

as '3.0' versions are updated thru 'Version.Details.xml'.

I will update:

- NUnit3TemplatesVersion
- MicrosoftDotNetCommonItemTemplatesPackageVersion
    
to '3.0' versioned.

Reply:
Well, but then... when we move to 3.0, how is this going to go? We will go with the duplicate package name with just different versions?

Reply:
We will have to do something with the 2 msbuild variables when moving from 3.0 to 3.1. 

Plan:
3.0 will move to a 'hard-coded' release version of the referenced 2 template versions
3.1 will take up the 'updating' 2 versions

Before:
    ```
<MicrosoftDotnetWinFormsProjectTemplates30PackageVersion>$(MicrosoftDotnetWinFormsProjectTemplatesPackageVersion)</MicrosoftDotnetWinFormsProjectTemplates30PackageVersion>
<MicrosoftDotNetWpfProjectTemplates30PackageVersion>$(MicrosoftDotNetWpfProjectTemplatesPackageVersion)</MicrosoftDotNetWpfProjectTemplates30PackageVersion>
```

After:
        <MicrosoftDotnetWinFormsProjectTemplates31PackageVersion>$(MicrosoftDotnetWinFormsProjectTemplatesPackageVersion)</MicrosoftDotnetWinFormsProjectTemplates31PackageVersion>
<MicrosoftDotNetWpfProjectTemplates31PackageVersion>$(MicrosoftDotNetWpfProjectTemplatesPackageVersion)</MicrosoftDotNetWpfProjectTemplates31PackageVersion><MicrosoftDotnetWinFormsProjectTemplates30PackageVersion>4.8.0-preview8.19359.1</MicrosoftDotnetWinFormsProjectTemplates30PackageVersion>
    <MicrosoftDotNetWpfProjectTemplates30PackageVersion>3.0.0-preview8.19359.5</MicrosoftDotNetWpfProjectTemplates30PackageVersion>